website/docs/r/vapp_vm.html.markdown

```diff
@@ -63,11 +63,11 @@ resource "vcd_vapp_vm" "web2" {
   memory        = 2048
   cpus          = 1
 
-  metadata {
-    role         = "web"
-    env          = "staging"
-    version      = "v2"
-    my_extra_key = "My extra value"
+  metadata = {
+    role    = "web"
+    env     = "staging"
+    version = "v1"
+    bla     = "foo"
```

Comment:
Nit: "bla" may have negative associations for someone, and it's nice to have a sample with underscores. Maybe better:
`my_key = "my value"`

Reply:
changed